pkg/storage/loaders/mto_common_solution_contractor.go

```diff
@@ -21,11 +22,11 @@ type mtoCommonSolutionContractorLoaders struct {
 
 // MTOCommonSolutionContractor is the singleton instance of all LoaderWrappers related to MTOCommonSolutionContractor
 var MTOCommonSolutionContractor = &mtoCommonSolutionContractorLoaders{
-	ByID:                NewLoaderWrapper(batchMTOCommonSolutionContractorGetByID),
+	ByID:                NewLoaderWrapper(MTOCommonSolutionContractorGetByCommonSolutionIDLoader),
 	ByCommonSolutionKey: NewLoaderWrapper(batchMTOCommonSolutionContractorGetBySolutionKey),
```

Comment:
Can you double check the comment on the type definition? it looks like it is referencing an incorrect function (maybe copy and paste error)